topi/python/topi/image/resize.py

```diff
@@ -0,0 +1,40 @@
+"""TVM operator input resize compute."""
+from __future__ import absolute_import
+import topi
+
+def resize(data, out_size, layout="NCHW", align_corners=False, mode="BILINEAR", weights=None):
+    """Perform resize operation on the data.
+
+    Parameters
+    ----------
+    inputs : tvm.Tensor
+        inputs is a 4-D tensor with shape
+        [batch, channel, in_height, in_width]
+        or  [batch, in_height, in_width, channel]
+
+    out_size: Tuple
+        Output resolution scale to
+
+    layout: string
+        either "NCHW" or "NHWC"
+
+    align_corners: Boolean
+        To preserve the values at the corner pixels
+
+    mode: string
```

Comment:
c.f.  https://www.tensorflow.org/api_docs/python/tf/image/resize_images

mode->method,
NN-> NEAREST_NEIGHBOR.

comment style for options

```python
method: {"BILINEAR", "NEAREST_NEIGHBOR"}
      Description string
```